scripts/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
     depends_on:
     - api
     logging:
-      driver: syslog
+      driver: none
```

Comment:
I know you need to change this on macOS, but doing it more broadly might need some thought. I don't _think_ this Compose file is used beyond local machine or Buildkite, but removing logging entirely is probably not the right thing to do - it means that `docker logs montagu_proxy_1` won't work. Would be better to simply remove the `logging` stanza altogether so it uses Docker's default logging driver.

Probably not a change for this ticket, either way!

Reply:
This commit was accidental, I didn't really want to change it.

Reply:
No worries!